modules/bmo-troubleshooting-latency-when-deleting-a-baremetalhost-resource.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ipi-install-troubleshooothing-latency-when-deleting-a-baremetalhost-resource_{context}"]
+
+= Troubleshooting latency when deleting a BareMetalHost resource
+
+When the Bare Metal Operator (BMO) deletes a `BareMetalHost` resource, Ironic deprovisions the bare-metal host. For example, this might happen when scaling down a machine set. Deprovisioning involves a process known as "cleaning". If cleaning doesn't succeed, the deletion of the `BareMetalHost` resource will take a long time, because it retries three times after a timeout. The deletion might not finish.
```

Comment:
```suggestion
When the Bare Metal Operator (BMO) deletes a `BareMetalHost` resource, Ironic deprovisions the bare-metal host with a process known as cleaning. If cleaning does not succeed, deleting the `BareMetalHost` resource takes considerable time and may not fully delete. The deleting process retries three times.
```
Would something like this work? I think the information is great; it just needs to be cleaned up a little and simplified for the reader.